apps/web/src/app/billing/organizations/billing-sync-api-key.component.html

```diff
@@ -1,117 +1,94 @@
+        </bit-callout>
+      </ng-container>
+
+      <div *ngIf="clientSecret && !showRotateScreen">
+        <p>{{ "copyPasteBillingSync" | i18n }}</p>
+        <label for="clientSecret">{{ "billingSyncKey" | i18n }}</label>
+        <div class="input-group">
+          <input
+            id="clientSecret"
+            class="form-control text-monospace"
+            type="text"
+            [(ngModel)]="clientSecret"
+            [ngModelOptions]="{ standalone: true }"
+            name="clientSecret"
+            disabled
+          />
+          <div class="input-group-append">
+            <button
+              type="button"
+              class="btn btn-outline-secondary"
+              (click)="copy()"
+              [appA11yTitle]="'copy' | i18n"
+            >
+              <i class="bwi bwi-lg bwi-clone" aria-hidden="true"></i>
+            </button>
           </div>
         </div>
+        <div class="small text-muted mt-2" *ngIf="showLastSyncText">
+          <b class="font-weight-semibold">{{ "lastSync" | i18n }}:</b>
+          {{ lastSyncDate | date : "medium" }}
+        </div>
+        <div class="small text-danger mt-2" *ngIf="showAwaitingSyncText">
+          <i class="bwi bwi-error"></i>
+          {{
+            (daysBetween === 1 ? "awaitingSyncSingular" : "awaitingSyncPlural") | i18n : daysBetween
+          }}
+        </div>
       </div>
-      <div class="modal-footer">
-        <button
-          type="submit"
-          class="btn btn-primary btn-submit"
-          [disabled]="form.loading"
-          *ngIf="!clientSecret || showRotateScreen"
-        >
-          <i
-            class="bwi bwi-spinner bwi-spin"
-            title="{{ 'loading' | i18n }}"
-            *ngIf="form.loading"
-          ></i>
-          <span>
-            {{ submitButtonText }}
-          </span>
-        </button>
-        <button
-          type="button"
-          class="btn btn-outline-secondary"
-          data-dismiss="modal"
-          *ngIf="!showRotateScreen"
-        >
-          {{ "close" | i18n }}
-        </button>
-        <button
-          type="button"
-          class="btn btn-outline-secondary"
-          *ngIf="showRotateScreen"
-          (click)="cancelRotate()"
-        >
-          {{ "cancel" | i18n }}
-        </button>
-        <button
-          type="button"
-          class="btn btn-outline-secondary"
-          *ngIf="clientSecret && !showRotateScreen"
-          (click)="rotateToken()"
-        >
-          {{ "rotateToken" | i18n }}
-        </button>
-      </div>
-    </form>
-  </div>
-</div>
+    </div>
+    <ng-container bitDialogFooter>
+      <button
+        type="submit"
+        class="btn btn-primary btn-submit"
+        [disabled]="loading"
+        *ngIf="!clientSecret || showRotateScreen"
+      >
+        <i class="bwi bwi-spinner bwi-spin" title="{{ 'loading' | i18n }}" *ngIf="loading"></i>
+        <span>
+          {{ submitButtonText }}
+        </span>
+      </button>
```

Comment:
We can use `bitButton` and `bitFormButton.

```suggestion
      <button
        type="submit"
        bitButton
        bitFormButton
        buttonType="primary"
        *ngIf="!clientSecret || showRotateScreen"
      >
        {{ submitButtonText }}
      </button>
```